doc/source/cluster/kubernetes/k8s-ecosystem/volcano.md

```diff
@@ -0,0 +1,319 @@
+# kind: PodGroup
+# metadata:
+#   creationTimestamp: "2022-12-01T04:48:18Z"
+#   generation: 2
+#   name: ray-test-cluster-1-pg
+#   namespace: test
+#   ownerReferences:
+#   - apiVersion: ray.io/v1alpha1
+#     blockOwnerDeletion: true
+#     controller: true
+#     kind: RayCluster
+#     name: test-cluster-1
+#     uid: b3cf83dc-ef3a-4bb1-9c42-7d2a39c53358
+#   resourceVersion: "4427976"
+#   uid: 9087dd08-8f48-4592-a62e-21e9345b0872
+# spec:
+#   minMember: 3
+#   minResources:
+#     cpu: "3"
+#     memory: 4Gi
+#   queue: kuberay-test-queue
+# status:
+#   conditions:
+#   - lastTransitionTime: "2022-12-01T04:48:19Z"
+#     message: '3/3 tasks in gang unschedulable: pod group is not ready, 3 Pending,
+#       3 minAvailable; Pending: 3 Undetermined'
+#     reason: NotEnoughResources
+#     status: "True"
+#     transitionID: 3956b64f-fc52-4779-831e-d379648eecfc
+#     type: Unschedulable
+#   phase: Pending
+```
+
+Because our new cluster requires more CPU and RAM than our queue will allow, even though we could fit one of the pods with the remaining 1 CPU and 2Gi of RAM, none of the cluster's pods will be placed until there is enough room for all the pods. Without using Volcano for gang scheduling in this way, one of the pods would ordinarily be placed, leading to the cluster being partially allocated, and some jobs (like [Horovod](https://github.com/horovod/horovod) training) getting stuck waiting for resources to become available.
+
+We can see the effect this has on scheduling the pods for our new RayCluster, which are listed as `Pending`:
+
+```shell
+kubectl get pods
+
+# NAME                                            READY   STATUS         RESTARTS   AGE
+# test-cluster-0-worker-worker-ddfbz              1/1     Running        0          7m
+# test-cluster-0-head-vst5j                       1/1     Running        0          7m
+# test-cluster-0-worker-worker-57pc7              1/1     Running        0          6m59s
+# test-cluster-1-worker-worker-6tzf7              0/1     Pending        0          2m12s
+# test-cluster-1-head-6668q                       0/1     Pending        0          2m12s
+# test-cluster-1-worker-worker-n5g8k              0/1     Pending        0          2m12s
+```
+
+If we dig into the pod details, we'll see that this is indeed because Volcano cannot schedule the gang:
+
+```shell
+kubectl describe pod test-cluster-1-head-6668q | tail -n 3
+
+# Type     Reason            Age   From     Message
+# ----     ------            ----  ----     -------
+# Warning  FailedScheduling  4m5s  volcano  3/3 tasks in gang unschedulable: pod group is not ready, 3 Pending, 3 minAvailable; Pending: 3 Undetermined
+```
+
+Let's go ahead and delete the first RayCluster to clear up space in the queue:
+
+```shell
+kubectl delete raycluster test-cluster-0
+```
+
+The PodGroup for the second cluster has moved to the `Running` state, as there are now enough resources available to schedule the entire set of pods:
```

Comment:
```suggestion
The PodGroup for the second cluster changed to the `Running` state, because enough resources are now available to schedule the entire set of pods:
```